### PR TITLE
Linting vertical-pod-autoscaler examples

### PR DESCRIPTION
**Pull request by achinthagunasekara**
Linting vertical-pod-autoscaler examples

### CODE REVIEW

File: vertical-pod-autoscaler/examples/hamster-deprecated.yaml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # This config creates a deployment with two pods, each requesting 100 millicores
 # and trying to utilize slightly above 500 millicores (repeatedly using CPU for
 # 0.5s and sleeping 0.5s).
-# It also creates a corresponding Vertical Pod Autoscaler that adjusts the requests.
+# It also creates a corresponding Vertical Pod Autoscaler that adjusts the,
```
Comment (bskiba) [inline vertical-pod-autoscaler/examples/hamster-deprecated.yaml]: Please remove the comma.

File: vertical-pod-autoscaler/examples/hamster-deprecated.yaml [hunk 2]
```diff
@@ -11,6 +13,7 @@ spec:
   selector:
     matchLabels:
       app: hamster
+
```
Comment (bskiba) [inline vertical-pod-autoscaler/examples/hamster-deprecated.yaml]: Is the blank line intentional?

Comment (achinthagunasekara) [reply vertical-pod-autoscaler/examples/hamster-deprecated.yaml]: I added that to get a visual seperate between two items

File: vertical-pod-autoscaler/examples/hamster-deprecated.yaml (hunk 1 above)
Comment (achinthagunasekara) [reply vertical-pod-autoscaler/examples/hamster-deprecated.yaml]: fixed :)

**Pull request merged**